Sources/Application/main.swift

```diff
@@ -1 +1,8 @@
-print("Hello")
\ No newline at end of file
+import SnowManLogic
+import Dictionary
+
+let dictionary = DictionaryManager() 
+let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
```

Comment:
Would be nice to have consistent naming conventions. Why not `dictionaryManager` ?